commands/build.go

```diff
@@ -412,12 +416,40 @@ func runBuild(ctx context.Context, dockerCli command.Cli, options buildOptions)
 }
 
 // getImageID returns the image ID - the digest of the image config
-func getImageID(resp map[string]string) string {
-	dgst := resp[exptypes.ExporterImageDigestKey]
-	if v, ok := resp[exptypes.ExporterImageConfigDigestKey]; ok {
-		dgst = v
+func getImageID(resp map[string]string, dockerUsingContainerdSnapshotter bool) string {
```

Comment:
`dockerUsingContainerdSnapshotter` should be `preferImageDigest`.

Reply:
Done, thanks.